### PR TITLE
Show non numeric traits and trait raw data on trait detail page

### PR DESCRIPTION
**Pull request by lukasmueller**
Description 
-----------------------------------------------------

Checklist 
---------------------------------------------------------------------------------------
- [ ] Refactoring only
- [ ] Documentation only
- [ ] Fixture update only
- [ ] Bug fix
  - [ ] The relevant issue has been closed.
  - [ ] Further work is required.
- [ ] New feature
  - [ ] Relevant tests have been created and run.
  - [ ] Data was added to the fixture
    - [ ] Data was added via a patch in `/t/data/fixture/patches/`.
  - [ ] User-Facing Change
    - [ ] The user manual in `/docs` has been updated.
  - [ ] Any new Perl has been documented using **perldoc**.
  - [ ] Any new JavaScript has been documented using **JSDoc**.
  - [ ] Any new _legacy_ JavaScript has been moved from `/js` to `/js/source/legacy`.

### CODE REVIEW

File: mason/breeders_toolbox/trial.mas
Comment (alockrow) [inline mason/breeders_toolbox/trial.mas:1]: Raw is written twice in the section description: "View raw phenotype raw data"

**Review by alockrow — CHANGES_REQUESTED** [review]
Looks good except for raw being written twice in the section description. I also wasn't able to find any non-numeric data to test it on. Also of note: the smiley shows up while the page is loading; not sure if that matters at all.

File: mason/breeders_toolbox/trial.mas
Comment (lukasmueller) [reply mason/breeders_toolbox/trial.mas:1]: Thank you for catching this, fixed!

**Comment by lukasmueller** [comment]
I think I fixed the issue with the limited traits in the menu by moving the pageSize argument to the correct location in the ajax call.

**Review by darekbienkowski — APPROVED** [review]
;-)

File: mason/breeders_toolbox/trial/trial_raw_data.mas [hunk 1]
```diff
@@ -0,0 +1,119 @@
+
+<%args>
+$trial_id
+</%args>
+
+<& /util/import_javascript.mas, classes => ['CXGN.BreedersToolbox.HTMLSelect'] &>
+
+<h3>Select Trait</h3>
+
+<div id="trait_selector_for_raw_data">:-)</div><br />
```
Comment (nmenda) [inline mason/breeders_toolbox/trial/trial_raw_data.mas:10]: maybe replace this with "Loading..." ?

**Review by nmenda — APPROVED** [review]

**Pull request merged**